setup.py

```diff
@@ -190,7 +190,7 @@ def run_tests(self):
             'osc2midi = osc2midi.main:main [osc2midi]',
         ]
     },
-    tests_require=['pytest'],
+    tests_require=['pytest','mock'],
```

Comment:
Keyword arguments do not have spaces around the equal sign. Read PEP-8 and/or use the `--explain` option with `pydocstyle` and it will become clear! :)